users/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt, requests
 
-# Create your views here.
+from django.shortcuts import render, redirect
+from django.views     import View
+from django.http      import JsonResponse
+from django.conf      import settings
+
+from users.models     import User
+
+class KakaoAPI:
+    def __init__(self, authorization_code):
+        self.authorization_code = authorization_code
+        self.token_url = "https://kauth.kakao.com/oauth/token"
+        self.user_url  = "https://kapi.kakao.com/v2/user/me"
+        self.redirect_url = "http://10.58.7.130:8000/users/kakao/callback"
+        
+    def get_access_token(self):
+        data = {
+            'grant_type': 'authorization_code',
+            'client_id': settings.KAKAO_APP_KEY,
+            'redirect_uri': self.redirect_url,
+            'code': self.authorization_code
+        }
+        response     = requests.post(self.token_url, data)
+        print(response.json())
+        access_token = response.json()['access_token']
+        return access_token
+        
+    def get_user_information(self, access_token):
+        header = {
+            "Authorization" : f'Bearer {access_token}',
+            "Content-type"  : "application/x-www-form-unlencoded;charset=utf-8"
+        }
+        
+        user_info = requests.get(self.user_url, headers = header)
```

Comment:
마찬자기로 `requests` 모듈을 사용하실 때에는 `timeout` 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 발생 합니다. timeout 옵션을 추가해 주세요.
```suggestion
        response     = requests.post(self.token_url, data, timeout = ..)
```

Reply:
네 추가하겠습니다.